### DISCUSSION TITLE
Comment comprenez-vous cette question / ce champ ?

### DISCUSSION THREAD

**Discussion by JenniferStephan** [betagouv/ma-cantine#1029] [Question / réponse]
hello, 

comment comprenez-vous le champ "mode de production" ds la création d'une cantine ? 

<img width="125" alt="Capture d’écran 2022-02-08 à 10 53 13" src="https://user-images.githubusercontent.com/36134318/152962733-058a4b5b-f18e-47be-b1e6-a1d238935ba4.png">

**Comment by gauthierandre** [comment]
En gros, je le vois ainsi : **cuisine centrale** : une site de production central qui produit pour plusieurs site de consommation ; **cuisine-site** : le site de production est le site de consommation.

**Reply by valeriemerle** [reply]
Avec une nuance qui est que certaines cuisines centrales sont aussi parfois des cuisines sur place! Elles livrent à des sites distants (les satellites) et à une cantine sur place (juste à coté). La vie n'est pas simple...

**Reply by gauthierandre** [reply]
Oui, c'est vrai 🤣

**Reply by gauthierandre** [reply]
Je suis en ligne avec une usagère et je me permets de rajouter une complexité supplémentaire qui concerne notre champ, du fait de sa lecture de l'outil d'import : il y a des cuisines "sites" qui sont des _cuisines sous dérogation à l’obligation d’agrément à la cuisine centrale_. Une cuisine est centrale quand elle exporte une part importante des repas avec un volume important de production. Ici, en gros, elle n'exporte pas assez, mais en même temps la part d'export est importante par rapport à la production. Je ne sais pas s'il est nécessaire de rendre la catégorie "cuisine site" plus inclusive, mais je trouvais ce retour intéressant à partager.

**Comment by alemangui** [comment]
La vision alternative (suggérée par @hfroot) : Je suis le gérant d'une cuisine où l'on sert des repas (site, satellite, etc). Je met _Cuisine centrale_ si la nourriture que je sers vient d'une cuisine centrale, et _Cuisine-site_ si je fais tout sur place.

Le label "Mode de production" peut signifier "_où est-ce que je produit ce que je mets dans les assiettes ?_". Surtout sachant que la loi EGAlim concerne les cantines qui servent des assiettes et non pas les cuisines centrales. D'ailleurs normalement les cuisines centrales n'ont aucune fonctionnalité qui les concerne sur la plateforme, vu que ce n'est pas elles qui télédéclarent. Une cuisine centrale publiée n'est d'aucun intérêt au grand public non plus, qui cherche plutôt les cantines ou ses enfants (ou eux mêmes) mangent. Donc je ne comprends pas pourquoi une cuisine centrale créerait un compte. :thinking:

**Reply by gauthierandre** [reply]
J'en profite repartager des retours que j'ai eu d'utilisateurs.
Sur ma cantine, on a pas mal de cuisines centrales qui créent un compte ; de facto, ce sont les CC qui ont les données des établissements pour lesquelles elles produisent. Je ne sais pas si c'est général, mais ça semble assez fréquent. 
Ensuite, on était tous d'accord sur le fait que ce sont les sites de consommation qui doivent apparaître et en ce sens, il n'y a pas besoin d'avoir un compte CC sur ma cantine. Dans l'idée, sur ma cantine, toutes ces structures de ce type devraient "disparaitre" au profit des établissements - sites. Or, c'est là qu'il y a un problème. 
Entre septembre et novembre, j'ai eu l'occasion d'échanger avec des gestionnaires en CC pour les inviter à importer leurs établissements, et c'est une option qui est soit rejetée soit, lorsqu'elle est acceptée, ne se traduit jamais dans les faits, car je n'ai plus de retours des interlocuteurs. En gros, on leur demande de revenir en arrière de la centralisation qu'ils ont mis en place, notamment dans le suivi des achats.

**Reply by valeriemerle** [reply]
Je complète la réponse d'André: la loi souhaite effectivement aller jusqu'aux assiettes servies ans les cantines. Mais quand elles sont approvisionnées par des cuisines centrales, ces cuisines satellites n'ont pas forcement les données d'appro mais gerent par ex les mesures de lutte contre le gaspillage. Je pense qu'il faudrait proposer une option "satellite" en plus des deux autres pour répondre au cas d'usage où l'ensemble des données seraient disponibles par cuisines satellites (un jour)...

**Reply by alemangui** [reply]
Merci André pour cet info. Une possibilité serait d'adapter la plateforme pour les cuisines centrales : la UI ne devrait pas les proposer de publier ou de télédéclarer, mais les inviter à créer des cantines satellites. Or, la plateforme aujourd'hui est seulement pensée pour des cantines de consommation, et ça peut se prêter à confusion. 

Je m'écarte un peu du sujet qui était à la base si la question "Mode de production" était claire (ama non) - mais ça veut la peine de commencer à réfléchir à une UI et fonctionnalités pour des cantines centrales qui pourraient les aider à passer à l'acte et créer des cantines satellites.

**Reply by gauthierandre** [reply]
Oui, partant pour qu'on puisse réfléchir à ce sujet de développer un parcours plus incitatif comme tu le décris.
Après, il peut y avoir un intérêt à avoir des cuisines centrales publiées : 1) parce qu'en parfois elle est aussi un site de consommation ; 2) _temporairement_ : à défaut d'avoir toutes les cuisines sites, ça peut intéresser l'usager. Même si, je suis entièrement d'accord, cet aperçu est une moyenne et en tant que tel n'est pas très représentatif de la réalité de ce qui est servi dans un établissement précis. Je ne suis pas très convaincu de cette seconde hypothèse de pertinence, mais je la précise parce que les CC sont identifiées par les utilisateurs, surtout quand elle est municipale et sert bcp d'établissements.

**Reply by alemangui** [reply]
Tout à fait d'accord, si une cantine centrale est aussi un site de consommation, ça veut dire qu'elle est aussi une cantine site, et donc a toute sa place dans la plateforme. Et merci, je ne savais pas que certaines cantines centrales sont identifiées par les convives, dans ce cas en effet leur publication serait intéressante.

**Comment by hfroot** [comment]
résolu avec https://github.com/betagouv/ma-cantine/pull/1099
